packages/transformers/pwa-manifest/src/PWAManifestTransformer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+// https://developer.mozilla.org/en-US/docs/Web/Manifest
+
+import {Transformer} from '@parcel/plugin';
+import type {MutableAsset} from '@parcel/types';
+
+import nullthrows from 'nullthrows';
+
+const getSrcHandler = (opt = {}) => (asset, dep) => {
+  dep.src = asset.addURLDependency(dep.src, opt);
+};
+
+const handleArray = (asset, list) => {
+  if (!Array.isArray(list)) {
+    return;
+  }
+  list.forEach(getSrcHandler().bind(null, asset));
+};
+
+const DEPS = {
+  icons: handleArray,
+  screenshots: handleArray,
+  serviceworker: getSrcHandler({
+    isEntry: true,
+    env: {context: 'service-worker'}
```

Comment:
1. is it right to use `{context: 'service-worker'}` here?
2. could you clarify when we should use `isEntry: true`? can't figure it out from the code.

Reply:
`isEntry` is used to prevent the filename from changing based on the content. Service workers in particular need to have a consistent filename between builds, or browsers will treat them as different applications.

`context: 'service-worker'` should be correct there, yes.